presto-main/src/main/java/io/prestosql/execution/SqlQueryExecution.java

```diff
@@ -361,16 +359,6 @@ public void addFinalQueryInfoListener(StateChangeListener<QueryInfo> stateChange
     }
 
     private PlanRoot analyzeQuery()
-    {
-        try {
-            return doAnalyzeQuery();
-        }
-        catch (StackOverflowError e) {
-            throw new PrestoException(NOT_SUPPORTED, "statement is too large (stack overflow during analysis)", e);
```

Comment:
I'm ok with this, but want to check with @martint and @electrum

Reply:
Is this being replaced elsewhere?